Packs/ProofpointEmailSecurity/ReleaseNotes/1_0_9.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Proofpoint Email Security Event Collector
+
+- Fixed an issue with the **proofpoint-es-get-last-run-results** command.
```

Comment:
```suggestion
Fixed an issue with the ***proofpoint-es-get-last-run-results*** command.
```

Reply:
It will not change because it is not related to the above contribution but to the commit from master.